### ISSUE TITLE
Support OAuth Refresh Tokens

### ISSUE THREAD

**Issue by hjacobs** [zalando-stups/zign#28]
We are using implicit flow with refresh tokens --- so support them.

**Comment by gargravarr** [comment]
Use the provided refresh token to request a new access token, so a browser window is not opened.

**Comment by hjacobs** [comment]
TODO: manage multiple refresh tokens. @sarnowski proposed to save both the `access_token` and `refresh_token` both in `tokens.yaml`. I think that makes sense.

**Comment by gargravarr** [comment]
@hjacobs @sarnowski refresh tokens are already saved in tokens.yaml

**Comment by hjacobs** [comment]
@gargravarr yes, but we need to use the `refresh_token` in `tokens.yaml` to support different scopes per token.
